po/common/mr.po

```diff
@@ -23,7 +23,7 @@ msgstr "mr"
 #. put here the language name ("French", "Japanese", etc)
 msgctxt ":langname"
 msgid ":langname"
-msgstr "भाषानाव"
+msgstr "Marathi"
```

Comment:
I am not quite sure which language `:langname` should be in. In production, we have some cases where the name is in the native language of the file, and others where it is in English. More importantly, I do not see `:langname` being used outside of loading the translations. Am I missing something, @stephanegigandet @maddingue?

Reply:
No use for Crowdin. @maddingue  ?

Reply:
We could remove it then. It was generated when we transformed I18N.pm to .po files but it's not used.

Reply:
Filed https://github.com/openfoodfacts/openfoodfacts-server/issues/1553